examples/example_HSC_DRP-Prod_start.yaml

```diff
@@ -0,0 +1,22 @@
+name: MUST_OVERRIDE
+parent_name: HSC_DRP-Prod
+spec_name: HSC_DRP-Prod_htcondor
+collection:
+    out: MUST_OVERRIDE
+#   campaign_source: HSC/raw/PDR2/WIDE,HSC/raw/PDR2/DEEP,HSC/raw/PDR2/UDEEP
+data:
+    prod_area: MUST_OVERRIDE
+    lsst_version: MUST_OVERRIDE
+#   butler_repo: '/repo/main+sasquatch_dev'
```

Comment:
If these commented line represent potential things that can still be added here, okay, but if they are old/outdated things, please remove?